sys/src/libufs/libufs.json

```diff
@@ -9,6 +9,8 @@
 			"block.c",
 			"cgroup.c",
 			"inode.c",
+			"libufs_subr.c",
+			"ffs_tables.c",
 			"sblock.c",
```

Comment:
The standard is that build tool recognize automagicly a file named build.json. I know you saw older format in what we used libfoo.json or cmd.json.

Reply:
What do we call the kernel build.json when we already have build.json?